### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by matmair**
This is a huge diff as it was auto-generated to fix a few styling problems:
- fix trailing whitespaces
- fix (missing) file endings
- Small PEP fixes in support files
- resort all imports to isort standard

All this will be done automatically if the devs have installed the dev-enviroment (see #3000)

<a href="https://gitpod.io/#https://github.com/inventree/InvenTree/pull/3041"></a>

### CODE REVIEW

File: InvenTree/InvenTree/management/commands/rebuild_models.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ def handle(self, *args, **kwargs):
         try:
             print("Rebuilding PartCategory objects")
 
+            # InvenTree imports
```
Comment (SchrodingersGat) [inline InvenTree/InvenTree/management/commands/rebuild_models.py:31]: Can we disable these messages appearing throughout the code?

**Comment by SchrodingersGat** [comment]
I like the cleanup, but could do without the addition of the import comments everywhere in the code. Can we disable that option with isort?

File: InvenTree/InvenTree/management/commands/rebuild_models.py (hunk 1 above)
Comment (matmair) [reply InvenTree/InvenTree/management/commands/rebuild_models.py:31]: Yes, they can be configured

**Comment by matmair** [comment]
@SchrodingersGat I removed the comments

**Review by SchrodingersGat — APPROVED** [review]

**Pull request merged**